src/commands/CamelRouteCommand.ts

```diff
@@ -0,0 +1,123 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License", destination); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict'
+
+import * as path from 'path';
+import * as fs from 'fs';
+import { TextDocument, window, workspace, WorkspaceFolder } from 'vscode';
+import { CamelJBangTask } from '../tasks/CamelJBangTask';
+
+import validFilename = require('valid-filename');
+
+export class CamelRouteCommand {
+
+	public static ID_COMMAND_CAMEL_ROUTE_JBANG = 'jbang.apache.camel.routes';
+	public static YAML_FILE_EXTNSION = 'yaml'; // TODO: change to 'camel.yaml' after upstream camel issue is released (see https://issues.apache.org/jira/browse/CAMEL-19112)
+
+	public async create(): Promise<void> {
+		const workspaceFolder = this.getWorkspaceFolder();
+		const providedName = await this.showInputBox(workspaceFolder);
+
+		if(providedName) {
+			const fileName = this.getFullName(providedName, CamelRouteCommand.YAML_FILE_EXTNSION);
+			const filePath = this.computeFullPath(workspaceFolder, fileName);
+
+			//
+			await new CamelJBangTask(workspaceFolder, fileName).execute();
+
+			// open created file in vscode text editor
```

Comment:
the comment doesn't add value, the next lines is quite straightforward to open in Text Editor, if it is now e can still extract it in a function with a name clarifying it. it is easier to keep function names up to date than comments